DQMOffline/Lumi/interface/MiniBaconDefs.h

```diff
@@ -0,0 +1,11 @@
+#ifndef DQMOFFLINE_LUMI_MINIBACONDEFS_H
+#define DQMOFFLINE_LUMI_MINIBACONDEFS_H
+
+#include <bitset>
+
+const unsigned int kNTrigBit = 128;
```

Comment:
hi @xmniu - would definitely prefer we just use the CMS standard trigger un-coding - but if that is not possible, lets debug this one

here - what happens when there are more triggers than 128?